Directory.Packages.props

```diff
@@ -9,13 +9,16 @@
     <PackageVersion Include="Aspire.Hosting.NodeJS" Version="$(AspireVersion)" />
     <PackageVersion Include="Aspire.Hosting.Python" Version="$(AspireVersion)" />
     <PackageVersion Include="Aspire.Hosting.SqlServer" Version="$(AspireVersion)" />
+    <PackageVersion Include="Bogus" Version="35.6.1" />
```

Comment:
Move under the `Testing Packages` comment (since I think that's really only relevant in tests)